app/src/main/java/org/simple/clinic/home/patients/PatientsTabUpdate.kt

```diff
@@ -123,7 +123,7 @@ class PatientsTabUpdate(private val isNotifyAppUpdateAvailableV2Enabled: Boolean
   ): Next<PatientsTabModel, PatientsTabEffect> {
     return when (event.appUpdateNudgePriority!!) {
       LIGHT, MEDIUM -> showAppUpdateAvailableDialog(model, event)
-      CRITICAL, CRITICAL_SECURITY -> noChange()
+      CRITICAL, CRITICAL_SECURITY -> next(model.appUpdateNudgePriorityUpdated(event.appUpdateNudgePriority), ShowCriticalAppUpdateDialog(event.appUpdateNudgePriority))
```

Comment:
Since we are updating the model in both cases with update priority. Do you think we can extract it out of the when block and pass in something like `updatedModel` to the functions? This can be done in a later PR as well. Please make a note of it if so.